src/etc/inc/captiveportal.inc

```diff
@@ -1866,6 +1887,14 @@ function captiveportal_remove_entries($remove) {
 	}
 	$query .= ")";
 	captiveportal_write_db($query);
+
+	if (xmlrpc_sync_get_details($syncip, $syncport, $syncuser, $syncpass, $carp_loop)) {
+		$rpc_client = new pfsense_xmlrpc_client();
+		$rpc_client->setConnectionData($syncip, $syncport, $syncuser, $syncpass);
+		$rpc_client->set_noticefile("CaptivePortalUserSync");
+		$rpc_client->xmlrpc_method('captive_portal_sync', array('op' => 'remove_entries', 'zone' => $cpzone, 'entries' => base64_encode(serialize($remove))));
```

Comment:
This line is very long, either move the `array(...` part to a new line, or define the array before using it. I know we prefer to avoid single use variables but in this case it would make it much easier to read/follow.